lib_tgvoip.cmake

```diff
@@ -807,6 +807,12 @@ if (NOT TGVOIP_FOUND)
                 WEBRTC_POSIX
                 WEBRTC_MAC
             )
+        elseif (FREEBSD)
```

Comment:
is this used by someone? I believe if you want to achieve the real effect, you should change Telegram/cmake/lib_tgvoip.cmake in the tdesktop repo.

Reply:
Oh, shi... That actually depends on cmake FREEBSD definition that you refused to include in https://github.com/desktop-app/cmake_helpers/pull/123 :(